webapp/src/routes/Accounts/index.js

```diff
@@ -133,60 +131,34 @@ const Accounts = ({ ual }) => {
     setLoading(false)
   }
 
-  const handleOnKeyDown = (event) => {
-    if (event.keyCode !== 13) return
-
-    handleOnSearch(accountName)
-  }
-
   useEffect(() => {
     const params = queryString.parse(location.search)
 
     if (!params.account) {
-      setAccountName('eosio')
-      handleOnSearch('eosio')
+      setFilters({ owner: 'eosio' })
+      handleOnSearch({ owner: 'eosio' })
 
       return
     }
 
-    setAccountName(params.account)
-    handleOnSearch(params.account)
+    setFilters({ owner: params.account })
+    handleOnSearch({ owner: params.account })
     // eslint-disable-next-line
```

Comment:
we can fix this eslint comment wrapping `handleOnSearch` function with useCallback, please take a look!